collectors/0/druid.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import calendar
+import json
+import re
+import subprocess
+import sys
+import threading
+import time
+
+from dateutil import parser as dtparser
+
+
+DRUID_METRICS_DIR = '/var/log/druid/metrics'
+DRUID_ROLES = [
+        'coordinator',
+        'overlord',
+        'historical',
+        'broker',
+        'realtime'
+        ]
+ 
+EVENT_REGEX = re.compile('Event \[(.*)\]')
+ILLEGAL_CHARS_REGEX = re.compile('[^a-zA-Z0-9\-_./]')
+
+
+def format_tsd_key(metric_key, metric_value, time_, tags={}):
+    def sanitize(s):
+        return ILLEGAL_CHARS_REGEX.sub('_', s)
+
+    expanded_tags = ''.join([' %s=%s'.format(sanitize(key), sanitize(value)) for key, value in tags.iteritems()])
```

Comment:
Your use of `string.format` seems to be incorrect, and results in the tags not printing correctly.  I tried running your code and got the following result
```python
>>> event_string = 'Event [{"feed":"metrics","timestamp":"2015-04-01T21:24:47.486Z","service":"historical","host":"10.4.112.207:8083","metric":"cache/total/averageBytes","value":0}]'
>>> report(event_string)
druid.cache.total.averageBytes 1427923487 0  %s=%s %s=%s %s=%s
```

I think you can fix this by replacing `' %s=%s'.format(sanitize(key), sanitize(value))` with  `' {0}={1}'.format(sanitize(key), sanitize(value))`

You use `string.format` correctly in the next line, so I guess you know this already.